### PR TITLE
Change dependencies to react 15.

### PR DESCRIPTION
**Pull request by joshgordon**
I changed `package.json` so that this depends on react 15. I was having trouble installing before when it was depending on a different version of react.

### CODE REVIEW

**Comment by kenwheeler** [comment]
Can you set it up to accept 14 & 15? Thanks so much!

**Comment by joshgordon** [comment]
Sure thing. I think that should do it.

File: package.json [hunk 1]
```diff
@@ -68,9 +68,9 @@
     "babel-standalone": "^6.4.4",
     "codemirror": "^5.12.0",
     "css-loader": "~0.9.0",
-    "react-addons-test-utils": "^15.0.1",
-    "react": "15.0.1",
-    "react-dom": "^15.0.1",
+    "react-addons-test-utils": "^15.0.0 || ^0.14.7",
+    "react": "15.0.1 || ^0.14.7",
```
Comment (igreulich) [inline package.json]: Shouldn't this be `"react": "^15.0.1 || ^0.14.7",` to keep in-step with `react-dom`, and `react-addons-test-utils`?

File: package.json [hunk 2]
```diff
@@ -68,9 +68,9 @@
     "babel-standalone": "^6.4.4",
     "codemirror": "^5.12.0",
     "css-loader": "~0.9.0",
-    "react-addons-test-utils": "^0.14.7",
-    "react": "0.14.7",
-    "react-dom": "^0.14.7",
+    "react-addons-test-utils": "^15.0.0 || ^0.14.7",
+    "react": "15.0.1 || ^0.14.7",
+    "react-dom": "^15.0.1 || ^0.14.7",
```
Comment (igreulich) [inline package.json]: Shouldn't all of `react`, `react-dom`, and `react-addons-test-utils` be `^15.0.1 || ^0.14.7`, so they all stay in-step?

**Comment by RinatMullayanov** [comment]
Please, accept these changes

**Comment by RinatMullayanov** [comment]
@kenwheeler @joshgordon When, in the NPM will release with these changes?

**Pull request merged**